libs/platform/src/lib/components/form/radio/radio.component.ts

```diff
@@ -0,0 +1,175 @@
+import {
+    Component,
+    ChangeDetectorRef,
+    EventEmitter,
+    Input,
+    Output,
+    Optional,
+    Self,
+    ViewChild,
+    ElementRef,
+    ChangeDetectionStrategy
+} from '@angular/core';
+import { RadioButtonComponent, stateType } from '@fundamental-ngx/core';
+import { ControlValueAccessor, NgControl } from '@angular/forms';
+import { FormFieldControl, InputSize } from '../form-control';
+
+@Component({
+    selector: 'fdp-radio-button',
+    templateUrl: './radio.component.html',
+    styleUrls: ['./radio.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    providers: [{ provide: FormFieldControl, useExisting: PlatformRadioButtonComponent, multi: true }]
+})
+export class PlatformRadioButtonComponent implements ControlValueAccessor {
```

Comment:
Once again, you havent addressed thing that I said before. You asked me to comment things in slack and I have added comment for this. 

Check e.g. RB implementation for radio button in material design, and comment I said in slack